srcpkgs/texlive-bibtexextra/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'texlive-bibtexextra'
+pkgname=texlive-bibtexextra
+version=2020.55376
+revision=1
+archs="noarch"
+build_style="fetch"
+build_helper="texmf"
+depends="texlive-core"
+short_desc="TeX Live - Additional BibTeX styles and bibliography databases"
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="GPL-2.0-or-later"
+homepage="http://tug.org/texlive/"
+distfiles="https://sources.archlinux.org/other/texlive/${pkgname}-${version}-src.zip"
```

Comment:
No upstream tarballs?

Reply:
Upstream does not release tarballs, as it is designed to be used with `tlmgr`. This is what alpine uses too, and what @q66 said to use.

Reply:
It is not super difficult to generate our own tarballs if needed, but I don't see any reason to create our own tarballs for as long as archlinux provides tarballs.

Reply:
ok